pom.xml

```diff
@@ -129,7 +129,7 @@
         <!-- @deprecated since 0.11 -->
         <httpclient.version>4.5.6</httpclient.version> <!-- kept for compatibility in 0.11.0-SNAPSHOT, remove after -->
         <commons-lang3.version>3.3.2</commons-lang3.version>
-        <groovy.version>2.4.15</groovy.version> <!-- Version 2.4.7 supported by https://github.com/groovy/groovy-eclipse/wiki/Groovy-Eclipse-2.9.1-Release-Notes; not sure what more recent will be -->
+        <groovy.version>2.5.8</groovy.version> <!-- Version 2.4.7 supported by https://github.com/groovy/groovy-eclipse/wiki/Groovy-Eclipse-2.9.1-Release-Notes; not sure what more recent will be -->
```

Comment:
Can you change the comment to say which groovy-eclipse (as per my comment on the PR previously).